### ISSUE TITLE
Does someone reproduct the result in paper with this pytorch version code?

### ISSUE THREAD

**Issue by fengshi-cherish** [kanezaki/pytorch-rotationnet#15]
I can't reproduct the results in modelnet40 in paper by this pytorch version code, by vgg16, the best result is Prec@1 92.125 Prec@5 99.250, by alexnet, the best result is Prec@1 91.750 Prec@5 98.625, even not reach the  mean value. Does author or someone  reproduct the result in paper? Can you tell me how to set the hyperparameters?

**Comment by kanezaki** [comment]
Hi, I guess you used `modelnet40v2png` directory which is actually a subset of ModelNet40. Please use `modelnet40v2png_ori4` instead. The accuracy reached 96.434% with 138 epochs when I used AlexNet as the base architecture with default parameters.
